.travis.yml

```diff
@@ -4,9 +4,12 @@ language: java
 # To use Docker's privileged mode, we need to enable sudo.
 sudo: required
 
-matrix:
-  include:
-    - jdk: openjdk11
+jdk:
+  - openjdk11
+
+env:
+  - DOCKER_DB_VERSION=7.0.2
+  - DOCKER_DB_VERSION=6.2.12
```

Comment:
why these 2 env vars have the same name?

Reply:
Those are variants. Travis will permutate them.